src/app/shared/services/workshops/user-workshop/user-workshop.service.ts

```diff
@@ -73,6 +73,14 @@ export class UserWorkshopService {
     return this.http.put('/api/v2/Workshop/Update', formData);
   }
 
+  /**
+   * This method update workshop status
+   * @param workshop: Workshop
+   */
+  updateWorkshopStatus(workshop: Workshop): Observable<object> {
+    return this.http.put('/api/v1/Workshop/UpdateStatus', workshop);
```

Comment:
payload is not really correct (it should be an object with id and status only
![image](https://user-images.githubusercontent.com/27493959/184657385-aeb39d6a-5743-4e92-831f-f68e211de2b9.png)